packages/jest-haste-map/src/crawlers/__tests__/node.test.js

```diff
@@ -68,6 +68,7 @@ jest.mock('fs', () => {
 });
 
 const pearMatcher = path => /pear/.test(path);
+const createMap = obj => new Map(Object.keys(obj).map(key => [key, obj[key]]));
```

Comment:
Random question from a stranger here:

Is `Object.entries` compiled / usable here?

```js
const createMap = obj => new Map(Object.entries(obj));
```

Reply:
I used it before but we don't include polyfills and we support Node 6 (which doesn't support it), so (I had to remove it)[https://github.com/facebook/jest/pull/6960/commits]. `Object.entries` was first available in Node 7.

Reply:
Nice! Thanks for the feedback.